gradle-plugin/src/gradleTest/java/com/squareup/anvil/plugin/LifecycleTest.kt

```diff
@@ -89,12 +92,168 @@ class LifecycleTest : BaseGradleTest() {
       )
     }
 
-    shouldSucceed("compileKotlin")
+    shouldSucceed("compileJava", withHermeticTestKit = true)
```

Comment:
Why do we check `compileJava` now?

Reply:
Out of an abundance of caution. 😄 

As its name suggests, the `cmmpilekotlin` task only goes as far as running the Kotlin compiler.  But this test is creating a condition where the files generated from a previous compilation would be missing.  It's one of those cases where the Kotlin task actually succeeds but the subsequent Java compilation task fails.  I'll add a clarifying comment.